### PR TITLE
Soc 199 Removing special styles for english language wikias

### PR DESCRIPTION
**Pull request by lizlux**
Before we were adding some extra formatting for english wikias, since we could predict the size and position of the elements. This was going to be too hard to maintain though, so that special handling is removed. 

This also adds the correct styling for email inputs, both in the facebook modal and in the WikiaForm class.

### CODE REVIEW

File: extensions/wikia/UserLogin/css/UserLoginFacebook.scss [hunk 1]
```diff
@@ -91,7 +70,8 @@ $badge-background-color: darken($color-page, 20%);
 		float: none;
 
 		[type=text],
-		[type=password] {
+		[type=password],
+		[type=email] {
```
Comment (kvas-damian) [inline extensions/wikia/UserLogin/css/UserLoginFacebook.scss]: I think it's good habit to use quotation marks ```"``` around attribute values.

File: extensions/wikia/UserLogin/templates/FacebookSignup_modalHeader.mustache [hunk 2]
```diff
@@ -1,3 +1,2 @@
 <h3>{{signupMsg}}</h3>
-<span>{{orMsg}}</span>
 <h3>{{loginMsg}}</h3>
```
Comment (kvas-damian) [inline extensions/wikia/UserLogin/templates/FacebookSignup_modalHeader.mustache]: It' not your change, but missing new line at the end of file ;)

File: extensions/wikia/WikiaStyleGuide/css/Form.scss [hunk 3]
```diff
@@ -23,67 +20,80 @@ $success-color: #3d8d42;
...
 		margin-bottom: 5px;
 		padding: 5px 7px;
-		input[type=text],
-		input[type=url],
-		input[type=password],
+
+		[type=text],
+		[type=url],
+		[type=password],
+		[type=email],
```
Comment (kvas-damian) [inline extensions/wikia/WikiaStyleGuide/css/Form.scss]: quotes again

**Comment by kvas-damian** [comment]
Left couple minor comments. Besides that LGTM

**Comment by lizlux** [comment]
Responded to @kvas-damian's comments. Merging.

**Pull request merged**